statsmodels/tools/tests/test_grouputils.py

```diff
@@ -314,24 +314,24 @@ def test_dummy_sparse():
     indi = dummy_sparse(g)
     assert isinstance(indi, sparse.csr.csr_matrix)
     result = indi.todense()
-    expected = np.matrix([[1, 0, 0],
-                          [1, 0, 0],
-                          [0, 0, 1],
-                          [0, 1, 0],
-                          [0, 1, 0],
-                          [0, 0, 1],
-                          [1, 0, 0]], dtype=np.int8)
+    expected = np.array([[1, 0, 0],
```

Comment:
in my scipy version `todense` returns a matrix and not an array. 
np.matrix is intentional, or at least it was at the time

Reply:
what scipy version?

Reply:
I'm using 0.18, but the current docs also say matrix is returned
https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.todense.html